Source/Model/Nodes/SVGMarker.swift

```diff
@@ -0,0 +1,166 @@
+#if os(WASI) || os(Linux)
+import Foundation
+#else
+import SwiftUI
+import Combine
+#endif
+
+public class SVGMarker: SVGNode {
+    public enum Orient {
+        case auto
+        case autoStartReverse
+        case angle(Float)
+
+        public init?(rawValue: String?) {
+            guard let rawValue else {
+                self = .angle(0)
+                return
+            }
+            if rawValue == "auto" {
```

Comment:
![CleanShot 2025-05-27 at 20 29 20@2x](https://github.com/user-attachments/assets/1f8abd88-275f-42e0-a67c-038133e981c4)

do we want to support rad?